Change IAM docs to focus on IAM v2 (#2715)
chef/automate: components/automate-chef-io/content/docs/teams.md

    weight = 60
+++

## Overview

> This guide will show you how to manage Chef Automate teams. Import existing teams into Chef Automate with [Microsoft AD (LDAP)]({{< ref "configuration.md#microsoft-active-directory" >}}), [generic LDAP]({{< ref "configuration.md#ldap" >}}) or [SAML]({{< ref "configuration.md#saml" >}}). You can create local Chef Automate teams that are independent of LDAP or SAML.

Reviewer: ```suggestion
This guide will show you how to manage Chef Automate teams. Import existing teams into Chef Automate with [Microsoft AD (LDAP)]({{< ref "configuration.md#microsoft-active-directory" >}}), [generic LDAP]({{< ref "configuration.md#ldap" >}}), or [SAML]({{< ref "configuration.md#saml" >}}). You can create local Chef Automate teams that are independent of LDAP or SAML.
```

> Permission on the `iam:teams` resource is required to interact with teams. Any user that is part of the `admins` team, or the `Administrator` policy will have this permission. Otherwise, [IAM v2 custom policies]({{< relref "iam-v2-guide.md#creating-custom-policies" >}}) can be created to assign this permission.

Reviewer: ```suggestion
Permission on the `iam:teams` resource is required to interact with teams. Any user that is part of the `admins` team or the `Administrator` policy will have this permission. Otherwise, [IAM v2 custom policies]({{< relref "iam-v2-guide.md#creating-custom-policies" >}}) can be created to assign this permission.
```

> Permission on the `iam:teams` resource is required to interact with teams. Any user that is part of the `admins` team or the `Administrator` policy will have this permission. Otherwise, [IAM v2 custom policies]({{< relref "iam-v2-guide.md#creating-custom-policies" >}}) can be created to assign this permission.

Reviewer 2: I do not think we should really be talking about a _resource_ here since resources are _personae non gratae_. Especially since the end of the paragraph references "this permission". But what permission? Something like this would help users understand a bit better: "...actions under `iam:teams`, which could include `iam:teams:get`, `iam:teams:list`, etc., or just `iam:teams:*` for full access".

Same concern about users below.

## Managing Local Teams

### Creating Local Teams
Navigate to _Teams_ in the **Settings** tab. Then use the **Create Team** button, which opens a helper window for entering the team's _Name_, a unique _ID_, and optionally assign the team to some _Projects_.

![Create Local Team](/images/docs/admin-tab-teams-list.png)

### Deleting Local Teams

> Navigate to _Teams_ in the **Settings** tab. Then open the menu at the end of the table row select **Delete Team**.

Author: ```suggestion
Navigate to _Teams_ in the **Settings** tab. Then open the menu at the end of the table row and select **Delete Team**.
```

### Adding Local Users to Teams

> To add local users to a team, navigate to _Teams_ from the **Settings** tab and locate the team. Navigate to the team's page, then use the **Add Users** button.

Reviewer: ```suggestion
To add local users to a team, navigate to _Teams_ from the **Settings** tab and locate the team. Navigate to the team's page, and then use the **Add Users** button.
```

#### Admins Team

> Chef Automate comes with an _Admins_ team, local users can be added directly to this team to assign them admin permissions.

Reviewer: ```suggestion
Chef Automate comes with an _Admins_ team. Local users can be added directly to this team, which assigns them admin permissions.
```
